_posts/0000-01-02-boldatum.md

```diff
@@ -0,0 +1,7 @@
+----------
+layout: slides
+title: "Welcome to my slides"
+----------
+
```

Comment:
"love this closet"
![5ii](https://user-images.githubusercontent.com/86401193/123225417-9ef0de00-d4ca-11eb-8498-b19f10607024.png)